### DISCUSSION TITLE
Database lost, what's the procedure to reuse existing UPLOAD_LOCATION

### DISCUSSION THREAD

**Discussion by weber8thomas** [immich-app/immich#9224] [Q&A]
Hi,

I lost my immich database and its backup by doing a wrong manipulation on my server. I still have my UPLOAD_LOCATION folder and a backup for it.

Before going in any wrong additional commands, I wanted to have some guidance regarding the reuse of that UPLOAD_LOCATION. Is it possible to force user-id assignation to match the previous IDs used? Do you have any procedure on similar scenario?

Many thanks, 

Thomas

**Comment by schuhbacca** [comment]
You won't be able to "reuse" the folder structure unfortunately. You will need to recreate the users, generate an api key for them, then upload from their folder in the old upload location using the cli. Specifying each users api key in the cli tool will upload into their individual user account.

**Reply by weber8thomas** [reply]
Thanks @schuhbacca, will pointing to their related library/ top folder where all the data will be enough using the cli to scan all their assets?

**Reply by schuhbacca** [reply]
Correct. You will point the CLI to their old user folder, and use their associated api key. That will upload all photos from their folder into their user space in immich.

**Reply by akostadinov** [reply]
But how do you keep the metadata then? `.xmp`? Shouldn't immich rescan the directory tree if the files are moved at the appropriate new path?
